src/sage/misc/dev_tools.py

```diff
@@ -547,10 +548,10 @@ def expand_comma_separated_names(obj):
                 obj = [G[name]]
             else:
                 # 1.b. object inside a submodule of sage
-                obj = find_objects_from_name(name, 'sage')
+                obj = find_objects_from_name(name, 'sage', include_lazy_imports=True)
                 if not obj:
                     # 1.c. object from something already imported
-                    obj = find_objects_from_name(name)
+                    obj = find_objects_from_name(name, include_lazy_imports=True)
 
```

Comment:
What is `include_lazy_imports=True` for here? Lines from L556 below seem to remove lazy-imported objects anyway.

Reply:
Only *deprecated* lazy imports are removed.
Other lazy imports are resolved in lines 595-596

Reply:
OK.